### ISSUE TITLE
Add sorting and checkboxes for managing publisher and publications authorities

### ISSUE THREAD

**Issue by jstirnaman** [BibApp/BibApp#91]
A-Z lists need better sorting and functionality when used for managing authorities. Example: http://experts.kumc.edu/publications/authorities?page=A 

Suggestions from frequent users:
    \* Let me sort by ISSN, so I can easily see the duplicates
    \* Add checkboxes or buttons so I can select multiple entries at a time to move up to the top area, where I can then establish one of the recs as the authority

**Comment by hading** [comment]
I agree that this whole thing needs some work. It's mostly a matter of getting the time to do it.

For what it's worth, my preference would be to make the whole thing work more through javascript if possible. Roughly - The list of stuff at the top would be it's own separate thing. The a-z list would be a javascript widget (hence sortable, addressing the first point). Going to a new letter would just repopulate the widget without affecting the top portion. Adding/removing authorities would just be javascript altering the two pieces of the view - no need to go back to the server until actually submitting (or if we do just very lightweight). And this would really address the second point - it'd more or less be clicking a button or link and then the entry would move up or down as appropriate.

Unlike some of Bibapp's other problems, this one would be easy to work on by just making an entirely new set of views and controller actions in parallel with the existing ones (which could go away when the new ones are finished, or remain as semi-deprecated - they might still be useful if something like my proposal doesn't work very well for accessibility, which it might not). 

I may try to do something like this in the relatively near future and see how it goes, so if you have any comments it'd be appreciated.

**Comment by hading** [comment]
actually I had forgotten that these are the pages I had already put in a javascript table. While I'd still like to do what I outlined, I bet that I can add sorting with almost no effort. I'll look into it.

**Comment by hading** [comment]
I have the sorting part done and will push to master soon. Fairly trivial to add this piece. I think accomplishing the rest really does call for a bit of a redesign, though, so this will have to do for now.

**Comment by hading** [comment]
Some other possible ideas:
- drag and drop between the two areas
- a bigger revamp: show things marked as authorities as main entries, but have those publications over which they are authoritie below them (so a slightly more treeish structure). Have button to set one of these free of its authority. Have drag and drop to put something under and authority. I actually think this could work really well except for the problem of not having two entries on the same screen when you want them to be. But maybe this could be overcome by having a 'staging area' that could temporarily hold entries, i.e. you could find something you wanted to put under authority, drag it to that area (repeat if desired) and then find your authority and drag the entries back onto the authority to finalize things. (Again, not very accessibility friendly, but maybe a nice slick solution aside from that.)
